hack/update-go-module-dependencies.sh

```diff
@@ -36,13 +37,21 @@ for x in vendor/github.com/openshift/*; do
 		# Replace the installer with our own fork below in this script.
 		vendor/github.com/openshift/installer)
 			;;
-
+    
+    vendor/github.com/openshift/assisted-service)
```

Comment:
That's because `github.com/openshift/assisted-service/api` and `.../models` are actually their own modules that need to be handled separately. the surrounding loop doesn't expect the modules to be nested like this so i had to exclude it here.